src/v5/types/observablearray.ts

```diff
@@ -474,27 +472,48 @@ const arrayExtensions = {
      * Without this, everything works as well, but this works
      * faster as everything works on unproxied values
      */
+;["concat", "indexOf", "join", "lastIndexOf", "slice", "toString", "toLocaleString"].forEach(
```

Comment:
missing `flat`, `includes` and  `copyWithin` (?) which will force it to go through the slow proxy path. Also prototype check

Reply:
Good points
`flat` - I will add this one to V5 in a similar manner to `flatMap` (with prototype check) 
`includes` - probably missing for the same reason as `find/findIndex` were missing - method list copy pasted from V4 (I think it would be nice to provide own impl in V4, but I don't want to introduce new API atm...same for `flatMap` or whatever is missing in V4...)
`copyWithin` - that would require some additional work I suppose, probably out of scope of this issue